tools/tests/test_check_version_bump.py

```
@@ -172,3 +177,153 @@ def test_main_returns_the_status_as_exit_code(repo, capsys, setup, expected):
+def test_uncommitted_move_out_of_the_zone_is_seen(repo):
+    """The working-tree half of the rename fix, which was unpinned: its sibling
+    commits the move, so it exercised only the base diff. This is the half the
+    pre-commit line in AGENTS.md actually runs, and dropping `--no-renames`
+    there left the whole suite green."""
+    _run(repo, "mv", "skills/a.md", "docs-a.md")     # staged, not committed
+    status, lines = cvb.check("main")
+    assert status == FAIL
+    assert any("skills/a.md" in line for line in lines)
+
+
+def test_git_stderr_reaches_the_operator(repo):
```

Ruff (0.16.1)

[warning] Unused function argument: `repo`

(ARG001)